pkg/clientcmdutil/configutil.go

```diff
@@ -0,0 +1,89 @@
+package clientcmdutil
```

Comment:
Let's try to not introduce new packages named util. 

See "Bad package names" in https://blog.golang.org/package-names

Reply:
Got it, renamed to clientcmd.

Also going to follow up by figuring out how pervasive these wrappers of clientcmd.Load(...) are and will file a community issue around duplicate code.